### ISSUE TITLE
Docker image for admin-console

### ISSUE THREAD

**Issue by ultimateboy** [google/exposure-notifications-server#653]
### TL;DR
A docker image should be provided for running the admin-console for the purpose of using `docker run` instead of using `go run` to run the console.

### Design

**Proposal**

Currently, the admin console is only usable with a `go run`. Due to the fact it does not have authentication, it's not meant to be deployed publicly. But for development and testing purposes, it is useful to deploy it using docker similarly to all the other services.

Due to the specifics of how the `./Dockerfile` is written with the `SERVICE` build-arg (and the fact that the admin-console requires the template files), I lean towards having a new dockerfile, either specifically for the admin-console or generic for all the tools.

**Alternatives considered**

We could enhance the current Dockerfile to also work with the tools. But the added complexity that would be required in the Dockerfile doesn't seem to provide us any benefit. I think it's more understandable to just have separate files.

**Comment by ultimateboy** [comment]
I didn't want to submit the PR with the new dockerfile until it was decided it was the correct direction to go, but I'll go ahead and drop the file I've been using myself here.
```
FROM golang:1.14 AS builder

RUN apt-get -qq update && apt-get -yqq install upx

ENV GO111MODULE=on \
  CGO_ENABLED=0 \
  GOOS=linux \
  GOARCH=amd64

WORKDIR /src
COPY . .

RUN go build \
  -trimpath \
  -ldflags "-s -w -extldflags '-static'" \
  -installsuffix cgo \
  -tags netgo \
  -o /bin/service \
  ./tools/admin-console

RUN strip /bin/service
RUN upx -q -9 /bin/service

FROM scratch
COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
COPY --from=builder /bin/service /bin/service
COPY --from=builder /src/tools/admin-console/templates /tools/admin-console/templates

ENV PORT 8080

ENTRYPOINT ["/bin/service"]
```

**Comment by sethvargo** [comment]
@ultimateboy I think we can update the main Dockerfile to accept the full path instead of assuming cmd/ to make this more flexible.

**Comment by mikehelmick** [comment]
We can also move admin console to cmd

**Comment by ultimateboy** [comment]
How would the template files be copied into the result container in either of those cases?

**Comment by sethvargo** [comment]
I supposed we'd always copy them. It's a few bytes on top of the already super-small image.
